docs/users/cli-usage.md

```diff
@@ -140,7 +140,8 @@ ddev config --project-type=drupal8 --docroot=web --create-docroot
 ddev start
 ddev composer create "drupal/recommended-project:^8"
 ddev composer require drush/drush
-ddev launch
+ddev drush site:install -y
+ddev drush uli
```

Comment:
The `ddev launch` really helps people who don't have a terminal that lets them click links. They like it a lot.
```suggestion
ddev drush uli
ddev launch
```